<a href="https://colab.research.google.com/github/hymachandhininamana/Basic-ML/blob/main/My%20first%20notebook%20in%20colab.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

                                               *   EVEN OR ODD

In [1]:
n=int(input())
if(n%2==0):
  print("even")
else:
  print("odd")
  

8
even


Explanation:

1. This program is used to find whether the given number is even or odd.
2. First let us consider an input n. 
3. Here 'int' represents the input that is an integer.
4. The condition for an integer to be an even number is "it must be divisible by
   2" and the condition for the odd number is "it not divisible by 2".
5. So, I have taken the condition that is if the number is divisible by two then
   the output will be even and if the number is not divisible by two then the 
   output will be odd.
6. In this program I have used 'if-else' condition.
7. "print" is used to give the output.
8. Here I have given input as 8. Since 8 is even number the output printed is even.

                                      * FACTORIAL OF A GIVEN NUMBER

In [4]:
n=int(input())
factorial=1
for i in range(1,n+1):
  factorial=factorial*i
print(factorial)


5
120


Explanation:
1. This program is used to find factorial of the given number.
2. I have used "for loop" in this program.
3. I have taken the required input as 'n'. Since the input is an integer, 'int'
   is used to represent it.
4. I have given factorial value as 1 and 1 is stored in it.
5. I have taken the range of i from 1 to (n+1).
6. Each value of i is multiplied with the factorial which wil be stored in the 
   factorial. 
7. Now use the print which is used to print the factorial of the given number 
   after the loop is ended.
8. Here I have given the input as 5. The factorial of Five is '120'.
9. Hence the output printed is 120.

                               * SUM OF FIRST 'N' NATURAL NUMBERS

In [7]:
 n=int(input())
 sum=0
 for i in range (1,n+1):
   sum=sum+i
   i=i+1
 print(sum)  

7
28


Explanation:

1. This program is used to find the sum of 'n' natural nubers.
2. "For loop" is used in this program.
3. Here 'n' is the input and 'int' represents that input is an integer.
4. Here for loop is runned in the range of i from 1 to (n+1).
5. 'sum' value is assigned as zero.
6. Every value of i is multiplied with the sum and is stored in sum
7. "print" is used to print the output.
8. Here I have given the input as 7.
9. Since the sum of first 7 natural numbers is 28.
10. So the output printed is 28.